web-ui/src/api/feedback.js

```diff
@@ -51,6 +51,12 @@ export const getFeedbackRequestById = async (id, cookie) => {
   });
 };
 
+export const getAllQuestionsAndAnswers = async (requestId, cookie) => {
+return resolve ({
+
+})
+}
+
```

Comment:
This seems...incomplete?

Reply:
I spoke to Julia. This was an unused empty api call. It has been removed.